config_manager.go

```diff
@@ -335,21 +262,16 @@ func (cm *ConfigManager) LogLevels() string {
 }
 
 func (cm *ConfigManager) RefreshInterval() RefreshInterval {
-	switch {
-	case cm.Config.RefreshInterval != nil && cm.Flags.RefreshInterval.Duration != 0:
-		return cm.Flags.RefreshInterval
-	case cm.Config.RefreshInterval != nil:
-		return *cm.Config.RefreshInterval
-	case cm.Flags.RefreshInterval.Duration != 0:
+	if cm.Flags.RefreshInterval != 0 {
 		return cm.Flags.RefreshInterval
 	}
-	return RefreshInterval{Duration: MinimumRefreshInterval}
+	return MinimumRefreshInterval
 }
```

Comment:
This doesn't consider `cm.Config.RefreshInterval`. `cm.Flags.RefreshInterval` should take priority if both are specified.